config/application.rb

```diff
@@ -1,33 +1,37 @@
-require File.expand_path('../boot', __FILE__)
+require_relative 'boot'
 
 require 'rails/all'
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+require_relative '../lib/web_socket_handler'
+
 module Community
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
```

Comment:
Did this comment come from the upgrade utility? It's not quite accurate here and could be misleading in the future, since the original Rails version for this app wasn't really 5.1.

Reply:
Yes, it did come from the upgrade utility.

Reply:
I think it's fine to leave it for now and see what happens on next upgrade